lib/spatial_features/importers/shapefile.rb

```diff
@@ -7,8 +7,9 @@ class Shapefile < Base
       class_attribute :default_proj4_projection
 
       def initialize(data, *args, proj4: nil, **options)
-        super(data, *args, **options)
-        @proj4 = proj4
+        @proj4 = options.delete(:proj4)
+        @shp_file_name = options.delete(:shp_file_name)
```

Comment:
Why do you need to delete `proj4` from `options`? Isn't it already extracted by the keyword args? We should be able to do the same with `shp_file_path`.

Reply:
Thanks for that reminder - I got a little twisted around with **options during my spelunking